core/src/main/java/org/apache/carbondata/core/datastore/page/statistics/PrimitivePageStatsCollector.java

```diff
@@ -243,6 +244,11 @@ private int getDecimalCount(double value) {
       int integerPlaces = strValue.indexOf('.');
       if (-1 != integerPlaces) {
         decimalPlaces = strValue.length() - integerPlaces - 1;
+        if (decimalPlaces == 1) {
```

Comment:
please add a comment to explain the scenario for which this code is added...the logic is clear but not sure for the scenario where it can lead to error/exception

Reply:
ok